### DISCUSSION TITLE
Practica de Pokedex - Lista

### DISCUSSION THREAD

**Discussion by fabiancgc12** [Launch-X-Latam/MisionFrontEnd#372] [General]
Buenas tardes, despues de varios dolores de cabeza con los estilos termino mi practica de pokedex. 
Algunos tips si quieren usarla:

- En el input pueden colocar desde un nombre hasta el id de la pokedex
- En el input pueden colocar algo como (numero)-(numero) para buscar un rango de pokemonsm, por ejemplo 25-30 te traera del 25 al 30 o 100-150 te traera del 100 al 150. una vez carguen pueden dar click en los mini sprites para cargar dicho pokemon.
- Los 3 botones de colores de arriba les permiten cambiar el theme de la pokedex
- Las flechas de izquierda y derecha les permiten navegar entre el pokemon anterior y el siguiente

Link del repositorio: https://github.com/fabiancgc12/MisionFrontEnd/tree/main/modulo%204%20-%20JS

Link de la pagina: https://fabiancgc12.github.io/MisionFrontEnd/modulo%204%20-%20JS/

**Comment by Rodrigorenteria** [comment]
Está increíble

**Comment by Cerecero** [comment]
Esta genial, a mi tambien se me ocurrio lo de agregar a MissingNo cuando no se ecuentra un pokemon jaja.

**Comment by JoseEduardoVelazquezCN** [comment]
Está increíble, felicidades 👍

**Comment by Z3us-77** [comment]
felicidades, esta super pro, imagino ya tienes equipo para hackaton, yo soy un principiante

**Comment by CGux8** [comment]
gracias por la humillacion jajajaja
